libbeat/processors/add_cloud_metadata/provider_google_gce_test.go

```diff
@@ -31,7 +31,10 @@ import (
 
 const gceMetadataV1 = `{
   "instance": {
-    "attributes": {},
+    "attributes": {
+		"kubeconfig": "apiVersion: v1\nkind: Config\nclusters:\n- cluster:\n    server: https://35.223.150.34\n",
+		"cluster-name": "staging-marketing-k8s"
+	},
```

Comment:
Could you add a test for an instance without these attributes?

Reply:
And, for both cases, maybe also a test for an event that already has the `orchestrator.cluster` values set, to check that they are not overriden.

Reply:
👍🏼 good catch!